packages/store/test/unit/secure-store/index.test.ts

```diff
@@ -1,7 +1,83 @@
+// import type { Logger } from '@sap-ux/logger';
+// import { keyring } from '@zowe/secrets-for-zowe-sdk';
+
+// jest.mock('../../../src/secure-store/key-store');
+// jest.mock('../../../src/secure-store/dummy-store');
+// jest.mock('../../../src/utils');
+// jest.mock('@zowe/secrets-for-zowe-sdk', () => ({
+//   keyring: {}
+// }));
+
+// describe('getSecureStore', () => {
+//   let log;
+
+//   beforeEach(() => {
+//     log = { warn: jest.fn(), info: jest.fn() } as unknown as Logger;
+//     jest.clearAllMocks();
+//   });
+
+//   // it('should use DummyStore when secure store is disabled by App Studio', () => {
+//   //   (utils.isAppStudio as jest.Mock).mockReturnValue(true);
+
+//   //   const store = getSecureStore(log);
+
+//   //   expect(store).toBeInstanceOf(DummyStore); // Check if DummyStore is returned
+//   //   expect(log.info).toHaveBeenCalledWith('Secure store disabled, using DummyStore.'); // Log assertion
+//   // });
+
+//   // it('should use DummyStore when secure store is disabled via environment variable', () => {
+//   //   process.env.FIORI_TOOLS_DISABLE_SECURE_STORE = 'true'; // Simulate environment variable
+
+//   //   const store = getSecureStore(log);
+
+//   //   expect(store).toBeInstanceOf(DummyStore); // Check if DummyStore is returned
+//   //   expect(log.info).toHaveBeenCalledWith('Secure store disabled, using DummyStore.'); // Log assertion
+//   // });
+
+//   it('should use KeyStoreManager when Zowe SDK is successfully loaded', () => {
+//     (utils.isAppStudio as jest.Mock).mockReturnValue(false);
+//     delete process.env.FIORI_TOOLS_DISABLE_SECURE_STORE; // Clear environment variable
+//     // Mocking the `keyring` object as if the SDK was successfully loaded
+//     jest.mock('@zowe/secrets-for-zowe-sdk', () => ({
+//       keyring: {
+//         getPassword: jest.fn(),
+//         setPassword: jest.fn(),
+//         deletePassword: jest.fn,
+//         findCredentials: jest.fn()
+//       }
+//     }));
+
+//     const store = getSecureStore(log);
+
+//     expect(store).toBeInstanceOf(KeyStoreManager); // Check if KeyStoreManager is returned
+//     expect(log.info).toHaveBeenCalledWith('Using KeyStoreManager for secure storage.'); // Log assertion
+//   });
+
+//   it('should fall back to DummyStore if Zowe SDK cannot be loaded', () => {
+//     (utils.isAppStudio as jest.Mock).mockReturnValue(false);
+//     delete process.env.FIORI_TOOLS_DISABLE_SECURE_STORE; // Clear environment variable
+//     jest.clearAllMocks();
+//     jest.resetAllMocks();
+//     // Mocking the `keyring` object as if the SDK was successfully loaded
+//     jest.mock('@zowe/secrets-for-zowe-sdk', () => ({
+//       keyring: null
+//     }));
+
+//     const store = getSecureStore(log);
+
+//     expect(store).toBeInstanceOf(DummyStore);
+//     expect(log.warn).toHaveBeenCalledWith('Falling back to DummyStore as secure storage could not be initialized.'); // Log assertion
+//   });
+// });
+
```

Comment:
uncomment or remove?

Reply:
removed all necessary comments and console logs..